src/main/java/com/commercetools/sync/types/utils/TypeUpdatePlainEnumActionUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.commercetools.sync.types.utils;
+
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.EnumValue;
+import io.sphere.sdk.types.Type;
+import io.sphere.sdk.types.commands.updateactions.AddEnumValue;
+import io.sphere.sdk.types.commands.updateactions.ChangeEnumValueOrder;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildAddEnumValuesUpdateActions;
+import static com.commercetools.sync.types.utils.TypeUpdateEnumActionsUtils.buildChangeEnumValuesOrderUpdateAction;
+import static java.util.Collections.emptyList;
+
+public final class TypeUpdatePlainEnumActionUtils {
+    /**
+     * Compares a list of old {@link EnumValue}s with a list of new {@link EnumValue}s for a given
+     * field definition.
+     * The method serves as a generic implementation for plain enum values syncing. The method takes in functions
+     * for building the required update actions (AddEnumValue, ChangeEnumValueOrder
+     * and 1-1 update actions on plain enum values (e.g. changeLabel) for the required resource.
+     *
+     * <p>If the list of new {@link EnumValue}s is {@code null}, then remove actions are built for
```

Comment:
but I don't see remove actions built here :( 

I think we can't build remove update actions in this case.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/9801dbd87fa39ffda80ead4f33a0fac5f7103070#diff-32d75125deae33b8dc07b973a10de669L28